bspump/ipc/datagram.py

```diff
@@ -32,18 +32,20 @@ def __init__(self, app, pipeline, id=None, config=None):
 		# Receive Buffer Size
 		self.ReceiveBufferSize = int(self.Config['receiver_buffer_size'])
 
-		if ":" in self.Address:
-			host, port = self.Address.rsplit(":", maxsplit=1)
-			(family, socktype, proto, canonname, sockaddr) = socket.getaddrinfo(host, port)[0]
+		for addrline in self.Address.split('\n'):
```

Comment:
There is no loop needed here, only one address could be specified.

Reply:
got it!

Reply:
We can implement that later, if there will be a demand for this ... currently, we need only support for space.
@mpavelka right?

Reply:
@ateska , right. Currently I don't think we will ever need to listen on multiple addresses defined by one pipeline's configuration.